src/Bundle/Twig/Token/JavascriptTokenParser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Hostnet\Bundle\WebpackBundle\Twig\Token;
+
+/**
+ * @author Harold Iedema <hiedema@hostnet.nl>
+ */
+class JavascriptTokenParser extends TokenParser
+{
+    /**
+     * Constant declaration for easy accessibility by the TwigParser component.
+     *
+     * @var string
+     */
+    const TAG = 'webpack_javascripts';
```

Comment:
I like the idea, but this would make the TwigParser imensly more complicated to detect split points :(

Also, images and plain CSS don't work. In my opinion this is kind of overkilling it, or as you already mentioned; over-engineering.

I'll see what I can do about the "js" and "css" options. Image is out of the question though. Thats just not how webpack works.